response_operations_ui/templates/components/search/_macro.njk

```diff
@@ -2,17 +2,19 @@
     {% from "components/input/_macro.njk" import onsInput %}
     {% from "components/button/_macro.njk" import onsButton %}
 
-    {% set labelPlaceholder = ' label--placeholder ' if params.accessiblePlaceholder %}
-    {% set inputPlaceholder = ' input--placeholder ' if params.accessiblePlaceholder %}
-    {% set inputClasses = params.input.classes if params.input.classes %}
+    {% set labelPlaceholder = ' label--placeholder ' if params.accessiblePlaceholder else "" %}
```

Comment:
doesn't it automatically get set to an empty string if it's not true?

Reply:
One for Matt Simon , I dont know how those files are generated